### PR TITLE
HRMP Channel Instructions Update

### LINKED ISSUE #4970: Update HRMP channel opening instructions
https://wiki.polkadot.network/docs/build-hrmp-channels

Make necessary changes to the doc by using the info from the latest referenda opening HRMP channels and also incorporating OpenGov pipeline. https://polkadot.polkassembly.io/referendum/123

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

**Comment by DrW3RK** [comment]
"Submission of the proposal as an external motion to Council" This section can be removed as well.

File: docs/build/build-hrmp-channels.md [hunk 1]
```diff
@@ -62,11 +62,11 @@ review and ultimately vote:
...
 2.  The use cases this channel will support for your chain;
 3.  Technical details of the proposal, including proposal parameters and technical details of this
-    call (On Kusama, most proposals were designed as a batchAll calls) :
+    call (On Kusama, most proposals were designed as a batchAll calls):
 
-    - A force transfer from Polkadot treasury to Statemint as deposit to accept and open an HRMP
-      channel with your chain;
-    - Send XCM message to Statemint to execute a transaction with superuser (root) permission.
+    - A `forceTansfer` of 20 DOT from the Treasury to Statemint sovereign account.
+    - A `forceOpenhrmpchannel` from Statemint (1,000) to ParaID.
```
Comment (CrackTheCode016) [inline docs/build/build-hrmp-channels.md:68]: I am a little confused - is this a general set of rules for HRMP openings? Or is it specific to the linked proposal only?

File: docs/build/build-hrmp-channels.md [hunk 2]
```diff
@@ -62,11 +62,11 @@ review and ultimately vote:
 1.  A request on what the proposal aims to do (opening an HRMP channel with Statemint);
 2.  The use cases this channel will support for your chain;
 3.  Technical details of the proposal, including proposal parameters and technical details of this
-    call (On Kusama, most proposals were designed as a batchAll calls) :
+    call (On Kusama, most proposals were designed as a batchAll calls):
 
-    - A force transfer from Polkadot treasury to Statemint as deposit to accept and open an HRMP
-      channel with your chain;
-    - Send XCM message to Statemint to execute a transaction with superuser (root) permission.
+    - A `forceTansfer` of 20 DOT from the Treasury to Statemint sovereign account.
```
Comment (CrackTheCode016) [inline docs/build/build-hrmp-channels.md]:
```suggestion
    - A `forceTransfer` of 20 DOT from the Treasury to Statemint sovereign account.
```

**Review by CrackTheCode016 — COMMENTED** [review]
It looks good to me, but maybe we should add a proper section for this example? it reads a little confusing.  

i can add this, and it can be reviewed!

**Comment by filippoweb3** [comment]
@CrackTheCode016 yes please add a section :) I will review it

**Review by CrackTheCode016 — APPROVED** [review]
@filippoweb3 It is good on my end, kindly check please when you can :) thank you!

**Pull request merged**